### DISCUSSION TITLE
L58 Chemistry Development Runs towards CAM7

### DISCUSSION THREAD

**Discussion by tilmes** [NCAR/amwg_dev#200] [TF1: Unified physics for L58/L93]
We are planning L58 full chemistry simulations to evaluate and develop the workhorse model for CAM7:
Here are experiments we want to perform and invite everyone to help to carefully evaluate them. Here are some ideas and questions:
**Model setup**: L58, ne30pg3, FCHISTL58, TS1 chemistry, CMIP6 emissions, OASISS DMS emissions
**Model code base:**  Latest development version, plus cam-dev (zm2), to include recent bug fixes, and water vapor fix
Need monthly mean output both interpolated to fv grid, and original output for budget analysis. 
All the following runs will be carefully evaluated (diagnostics and tuning will be applied for all MAM5 runs):
1. Run model with MAM4, orographic GW on, possibly try a test run with nudged QBO 
2. Like 1. but with MAM5 instead  -> Perform Tuning for MAM5 with chemistry
4. Like 2. but include updated chemistry  
5. Like 3. but include Upper Boundary Conditions
6. Like 4. but add PUMAS and CLUBB externals (depending on when this is done)
7. Like 5 but with new dust emissions
8.  Other developments as they become available. 

Please comment on what model version we should use and if you have other experiments you want us to add.
Current list of planned diagnostics, it would be great if we could add those in the standard diagnostic set as well.
AMWG standard plus chemistry and aerosols
Old AMWG diagnostics for more detailed chemistry / WACCM diagnostics
AOA diagnostics 
Tropospheric ozone
Ozone Hole Diagnostic 
Tape recorder
Budgets (chemistry / aerosol)
TEM diagnostics
QBO

**Comment by adamrher** [comment]
Can we wait until MAM5 is on the head of the trunk? Then I can make a branch where we deactivate PUMAS until we are ready to introduce it. And it will have the latest/greatest CLUBB externals (It is easier from an engineering perspective to then get our CLUBB externals in order before these MAM5 experiments). The sequence proposed by @JulioTBacmeister here: https://github.com/NCAR/amwg_dev/discussions/193 is inline with what I'm proposing.

**Reply by tilmes** [reply]
I would prefer to chemistry runs before the CLUBB externals get moved in, and then compare that run with the new CLUBB externals to make sure we are not missing up the chemistry.

**Comment by JulioTBacmeister** [comment]
My proposal is this:
We perform a series of F-case controls with the "26" configuration ASAP - 10, 20 years each. We'd enhance aerosol/chemistry output as needed by SImone and others.  Using Simone's compset style the runs would be

1) FCHISTL58 (need to discuss non-oro gw on/off)
2) FHISTL93
3) FCHISTL93 

I think we can and should start (2) immediately. If there are no issues with chemical BCs we could also start (3). I'd only hesitate because it may be good to look at our QBO and stratosphere in (2) before proceeding with a full chemistry run. 

Run (3) will provide a baseline to track impacts of the changes we will be making to CAM over the next few months.

**Reply by adamrher** [reply]
> We'd enhance aerosol/chemistry

As the goal is to provide a baseline of our 26 series, but in the f-cases, I take "enhance" to mean output aerosol diagnostics, and *not* incorporate MAM5 into these baselines.

**Comment by JulioTBacmeister** [comment]
Yes I meant aerosol/chemistry *output*

**Comment by adamrher** [comment]
@JulioTBacmeister I just spoke with @tilmes. There is a strong desire by ACOM to start testing incrementally the impacts of CLUBB, PUMAS, MAM5, specific UB's, organic aerosols, dust changes, etc ... w/ interactive chemistry. Simone is proposing a set of FCHISTL58 experiments (i.e., full chemistry):

1. 26c + MAM5 (Cecile's sandbox, includes SourceMods for restart bug fixes, and the two reference profile fixes to the SE dycore). 
2. add new CLUBB externals (move to head of the trunk; force off PUMAS, emitb4turb)
3. add new PUMAS
4. add emitb4turb
5. other stuff (see top of thread for list)

This is a bit slower than we planned on moving; we were planning on the next set of f-case experiments (after we run the 26 series baselines) being the head of the trunk, thereby shocking us with at least three new enhancements (CLUBB, PUMAS, emitb4turb), then bring in the aerosol enhancements. That was a fine plan IMO, we are already committed to these enhancements. We need to learn to live with them, and we already have an idea of what they each do individually. If ACOM wants to quantify the impact on chemistry each individual enhancement, that's fine by me as well.

**Reply by tilmes** [reply]
@adamrher I would run 1. but with MAM4  and then go to the latest trunk version (when MAM5 is hopefully soon included since it is very straight forward to add) and start 2, but first also without CLUBB externals, and just with MAM5.

**Reply by adamrher** [reply]
unfortunately, we can't run the old clubb externals on the head of the trunk.

**Reply by tilmes** [reply]
OK, maybe that is fine.

**Reply by adamrher** [reply]
New potential proposal for experiments w/ MAM5:

FCHISTL58:
1. 26c (cecile's sandbox)
2. 26c + MAM5 (cecile sandbox)
3. CLUBB + PUMAS + emitb4turb + MAM5 (head of trunk)**

**exploration of impacts of individual changes if needed. The reason we advocate for only doing the incremental approach if necessary is because these atmospheric enhancements are uncontroversial.

**Reply by tilmes** [reply]
Due to earlier simulations with emitb4turb, we saw some big changes in chemistry. But, we can first do the run with all the changes, and  later track back if needed.

**Reply by tilmes** [reply]
@adamrher We don't have MAM5 with chemistry available in Cecile's sandbox, and it would be much easier to use the new development code to run MAM5 ones it is included, so I don't have to make a new sandbox with MAM5 included based on Cecile's sandbox. Therefore it would be best to just start with the development version, when MAM5 is included. I can do a MAM4 run with Cecile's code first with chemistry. So:
1. 26c (Cecile's sandbox with the orographic GW).
2. MAM5 and head of the trunk  (without emitb4turb, and without PUMAS)   
3. MAM5 (head of the trunk)
I can also do first 3. and see what is different, but I suppose we do want to see what PUMAS and emitb4turb may do

**Comment by fvitt** [comment]
@tilmes, Please note that @cacraigucar is proposing using "LT" and "MT" in the compset names to indicate "low-top" 58 layers and "medium-top" 93 layers, respectively. See https://github.com/ESCOMP/CAM/issues/692

So, CAMChem compsets will have names like:
 FCLTHIST -- CAMChem with 58 vertical layers
 FCMTHIST -- CAMChem with 93 vertical layers

**Reply by cacraigucar** [reply]
We are proposing "LT" and "MT" for lowtop (58) and midtop (93)   We thought WACCM and/or WACCMX might not want us using "HT" for hightop.  This can certainly be discussed in https://github.com/ESCOMP/CAM/issues/692

**Reply by fvitt** [reply]
@cacraigucar Thanks for correcting me.  I adjusted my comment accordingly.

**Reply by tilmes** [reply]
That's fine with me

**Comment by dan800** [comment]
Agree that high-top is always how we refer to WACCM in the literature - ie the high-top version of CESM. So would not recommend using HT. 

Personally I like the L58 & L93 designations - adds more information. However, LT and MT would also work. 

Dan

**Comment by tilmes** [comment]
@JulioTBacmeister @cecilehannay @dkin 
Would anyone mind if we do our initial chemistry simulations for the 26c case from 1995-2010 instead of from 1980-2000 for now? We have more data to compare with if we run the later period? This run is not meant to be used for prescribing ozone, so maybe we can safe some computer and run time that way?
